src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.ts

```diff
@@ -110,6 +131,7 @@ export class TutorExerciseDashboardComponent implements OnInit {
         );
     }
 
+    // TODO CZ: too much duplicated code
```

Comment:
should we leave todos like that in the code?

Reply:
it is not ideal, but some of these TODOs describe lower priority tasks that we want to finish in the future, so better merging these TODOs now than loosing a lot of time.